cmd/lakefs-loadtest/cmd/entry.go

```diff
@@ -26,12 +25,13 @@ const createEntryPathLength = 110
 var entryCmd = &cobra.Command{
 	Use:   "entry <ref uri>",
 	Short: "Load test database with create entry calls",
-	Args: cmdutils.ValidationChain(
-		cobra.ExactArgs(1),
-		cmdutils.FuncValidator(0, uri.ValidateRefURI),
-	),
+	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		u := uri.Must(uri.Parse(args[0]))
+		if !u.IsRef() {
+			fmt.Printf("Invalid 'ref': %s", uri.ErrInvalidRefURI)
+			os.Exit(1)
+		}
```

Comment:
`MustRef...`

Reply:
The MustXXX functions are based on the parse with default where we use the lakectl specific environment variable.
In this case, we do not use it - this is part of `lakefs-loadtest` command line.
If we will have multiple command-line tools that use the same technic, I'll extract this functions out to `uri` package and pass the base URI value on each call.